doc/org/src/fit_model.jl

```diff
@@ -13,8 +13,10 @@ df_test = df[test_rows, :]
 mse(y1, y2) = mean((y1 - y2) .^ 2)
 
 regression_full = lm(@formula(CompileTime ~ ((Expected_insns + Expected_insns ^ 2) +
```

Comment:
Here begins the formula I have used for my model, it had MSE=0.7 on a 10-90% split.